crates/hdx_ast/src/css/values/fonts/types.rs

```diff
@@ -1 +1,79 @@
-pub(crate) use crate::css::units::*;
+use hdx_atom::atom;
+use hdx_lexer::Cursor;
+use hdx_parser::{diagnostics, Build, CursorStream, Parse, Parser, Peek, Result as ParserResult, ToCursors, T};
+
+mod func {
```

Comment:
This is, by convention, `mod kw`.

but is probably get rid of all the keywords and just peek an ident.